packages/storybook/src/stories/grid-rows.stories.tsx

```diff
@@ -513,7 +515,7 @@ export function EditRowsControl() {
         apiRef.current.updateRows([cellUpdate]);
         apiRef.current.publishEvent(
           GridEvents.cellEditStop,
-          apiRef.current.getCellParams(id, field),
+          apiRef.current.getCellParams(id, field) as GridCellEditStopParams,
```

Comment:
Is `reasons` really defined here ? 
Otherwise, can't it cause problem to let typescript think we are passing it to `cellEditStop` when we aren't ?

Reply:
I made `reason` optional. It's only available if the new API is used.